pkgs/top-level/aliases.nix

```diff
@@ -318,6 +318,14 @@ mapAliases ({
   xlibs = xorg; # added 2015-09
   youtubeDL = youtube-dl;  # added 2014-10-26
 
+  # added 2018-10-16
+  # TODO(ekleog): remove after 19.03 branch-off
+  # TODO(ekleog): add ‘wasm’ alias to ‘ocamlPackages.wasm’ after 19.09
+  # branch-off
+  wasm = lib.warn
+    "‘wasm’ package has been renamed ‘proglodyte-wasm’, and will be dropped in the next release"
+    proglodyte-wasm;
```

Comment:
You can just write: `wasm = proglodyte-wasm;` this will already generate the right warning message asking users to migrate.

Reply:
This hasn't when I tested with `nix-shell` :/

Reply:
mhm. Maybe this is only the case for `nixos/modules/rename.nix`

Reply:
@ryantm  do you have insights here?